kotlin/codegen/src/main/java/com/squareup/moshi/kotlin/codegen/api/AdapterGenerator.kt

```diff
@@ -234,7 +260,7 @@ internal class AdapterGenerator(
         if (property.hasConstructorDefault) {
           val inverted = (1 shl maskIndex).inv()
           result.addComment("\$mask = \$mask and (1 shl %L).inv()", maskIndex)
-          result.addStatement("%1L = %1L and %2L", maskName, inverted)
+          result.addStatement("%1L = %1L and %2L", maskNames[maskNameIndex], inverted)
```

Comment:
It'd be nice to emit `inverted` in hex? `0x80` for bit 4 instead of 16?

Reply:
b2b0a38